packages/core/src/ceramic.ts

```diff
@@ -569,8 +570,8 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
     }
 
     if (EnvironmentUtils.useRustCeramic()) {
-      this._logger.warn(
-        `Running Ceramic using rust-ceramic. If you want to use Ipfs, run with IPFS_FLAVOR=go.`
+      this._logger.imp(
+        `Running Ceramic using ceramic-one (rust-ceramic). To use IPFS (kubo), specify a --ipfs-url or run with IPFS_FLAVOR=go.`
```

Comment:
Should this be `and` since `IPFS_FLAVOR` has to be specified to run with Kubo regardless of whether the URL is specified?
```suggestion
        `Running Ceramic using ceramic-one (rust-ceramic). To use IPFS (kubo), specify a --ipfs-url and run with IPFS_FLAVOR=go.`
```

Reply:
Right now it actually isn't required. IPFS_FLAVOR only matters for the binary with these changes, and if you're running a remote node we simply log a warning if they don't match and change the value so the rest of the process does recon feed/pubsub appropriately.

Sorry, I pretty much swapped everything else out from under you because I didn't like how required parameters were being sent is as undefined.